cmd/kubeadm/app/phases/kubelet/flags.go

```diff
@@ -57,8 +56,10 @@ func WriteKubeletDynamicEnvFile(cfg *kubeadmapi.ClusterConfiguration, nodeReg *k
 		pauseImage:               images.GetPauseImage(cfg),
 		registerTaintsUsingFlags: registerTaintsUsingFlags,
 		execer:                   utilsexec.New(),
-		pidOfFunc:                procfs.PidOf,
-		defaultHostname:          hostName,
+		isProcessRunningFunc: func(name string) bool {
+			return utilsexec.New().Command("pidof", name).Run() == nil
```

Comment:
@thockin  - would this fit into the kernel package we discussed under https://github.com/kubernetes/utils/pull/99#discussion_r303085545 ?